tests/UnitTests/UIKit/UIImageTests.mm

```diff
@@ -0,0 +1,82 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Starboard/SmartTypes.h>
+#import <TestFramework.h>
+#import <UIKit/UIImage.h>
+
+static NSString* getPathToFile(NSString* fileName) {
+    static StrongId<NSString*> refPath = []() {
+        char fullPath[_MAX_PATH];
+        GetModuleFileNameA(NULL, fullPath, _MAX_PATH);
+        return [@(fullPath) stringByDeletingLastPathComponent];
+    }();
+    return [refPath stringByAppendingPathComponent:fileName];
+}
+
+template <typename VerifierLambda> // Takes the form void(*)(UIImage*,UIImage*)
+static void testRepresentationOfCroppedImage(NSString* source1, NSString* source2, CGRect cropRegion, VerifierLambda&& verify) {
```

Comment:
I am so not okay with checking that perceptually-encoded data is the same across launches of an application!